docs/user-documentation/metadata.md

```diff
@@ -1,15 +1,17 @@
 # Metadata in Islandora 8
 
-> TL;DR: In Islandora 8 metadata values are stored in _fields_ attached to _entities_ (objects) which are then serialized as JSON-LD before being submitted to Fedora and/or indexed in a triple-store.
+> TL;DR: In Islandora 8, metadata is stored in _fields_ attached to _entities_ (nodes or media). Metadata values are serialized as RDF in the form of JSON-LD before being submitted to Fedora and/or indexed in a triplestore.
 
 !!! note "Drupal 8 Terminology"
-    In Drupal 8, Fields can be attached to _entity sub-types_ (e.g. Content types, Vocabularies) or _entities_ (Users, Files). For more on Fields, see ["2.3 Content Entities and Fields"](https://www.drupal.org/docs/user_guide/en/planning-data-types.html) and ["6.3 Adding Basic Fields to a Content Type"](https://www.drupal.org/docs/user_guide/en/structure-fields.html) in the Official Guide.
+    In Drupal 8, Fields can be attached to _entity sub-types_ [(formerly called _bundles_](https://www.drupal.org/project/drupal/issues/1380720) (e.g. Content types, Media types, Vocabularies) or _entities_ (Users, Files). For more on Fields, see ["2.3 Content Entities and Fields"](https://www.drupal.org/docs/user_guide/en/planning-data-types.html) and ["6.3 Adding Basic Fields to a Content Type"](https://www.drupal.org/docs/user_guide/en/structure-fields.html) in the Official Drupal Guide.
```

Comment:
Hmmm, 'bundles' is still the official terminology. The issue listed is to _perform usability testing_ on the change rather than implement it (and the ticket is still unassigned). Perhaps this ticket can be put in a comment so we can revisit it later when we revise documentation again:
```
<!-- Next revision: check status of changing 'bundles' to 'entity sub-types' (https://www.drupal.org/project/drupal/issues/1380720). -->
```
That way the docs will reflect practice as it currently stand and won't confuse current readers but we are still aware of the possible change when we look at the source markup.

Reply:
So, at the docs _stood_ they said "entity sub_types". My first thought was "hey, that should be bundles!" but then I looked it up and yeah there's been a push since 2011 to change it to "entity sub-types". Thanks, i'll revise, and good suggestion on the comment.

Reply:
Marking resolved - current state is:
`Fields can be attached to _bundles_ (sometimes called _entity sub-types_ -- e.g. Content types, Media types, Vocabularies)`